src/main/kotlin/net/atos/zac/app/admin/ZaakafhandelParametersRestService.kt

```diff
@@ -73,12 +73,12 @@ class ZaakafhandelParametersRestService @Inject constructor(
     }
 
     /**
-     * Retrieve all CASE_DEFINITIONs that can be linked to a ZAAKTYPE
+     * Retrieve all case definitions that can be linked to a ZAAKTYPE
      *
-     * @return LIST of CASE_DEFINITIONs
+     * @return list of all case definitions
      */
     @GET
-    @Path("caseDefinition")
+    @Path("case-definitions")
```

Comment:
We prefer EN for all our code we agreed. However the very specific word `zaak` is special. Also because the word `case` is, although related, a little bit different you could argue. Technically I guess you could say that a zaak is a very specific kind of case.

Reply:
In this case (pun intended) however `case` is the right word because this endpoint is specifically about CMMN case definitions. It is _not_ about zaak definitions (such a thing does not even exist).